lib/reconcile/vernemq.go

```diff
@@ -297,7 +283,7 @@ func getVerneMQEnvVars(statefulSetName string, cr *apiv1alpha1.Astarte) []v1.Env
 
 func getVerneMQPodSpec(statefulSetName, dataVolumeName string, cr *apiv1alpha1.Astarte) v1.PodSpec {
 	serviceAccountName := statefulSetName
-	if pointy.BoolValue(cr.Spec.RBAC, false) {
+	if !cr.Spec.RBAC {
```

Comment:
Not equivalent to the old logic. See above.

Reply:
Just like the other one